test/module_loading/getNewArgvWithArguments.js

```diff
@@ -0,0 +1,15 @@
+const { rootOptionsDefaults } = require('../../constants');
+
+function getNewArgvWithArguments() {
+    return {
+        username: 'testbot',
```

Comment:
this is currently a repetition of /test/assignConfigArguments.js

should we always import defaults from same source (for example from the real getArgv.js as in #358 , or by creating a common shared /test/testDefaults.js where testing's username, apikey, etc. are declared once and required in all modules that use it?

also, currently we dont import all defaults from the real getArgv, only rejectnewmsg, current tests, especially in checkChallenge.test.js should be slightly updated to match defaults (for example it's unlikely to specifically declare an option as undefined if it has a default (ex: `config.minmaintimeblitz = undefined` which tests `--minmaintimeblitz undefined`)

this may be done in a separate PR rather